### PR TITLE
chore: temp kubefed fork

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

creating a temporary kubefed fork to use v1 cert-manager resources.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by joejulian — APPROVED** [review]

**Comment by hectorj2f** [comment]
@alejandroEsc Can you explain as part of the description why we need to add a kubefed fork ?

**Comment by alejandroEsc** [comment]
> @alejandroEsc Can you explain as part of the description why we need to add a kubefed fork ?

Yes the us-ksphere team needs a temporary and only temporary version of kubefed to test v1 cert-manager changes I made upstream. We need this for now and like i said wee will not use this any longer when the changes upstream stick.

**Comment by alejandroEsc** [comment]
helm2 test will fail, wrong api version of course.

File: staging/kubefed/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-home: https://github.com/kubernetes-sigs/kubefed
+apiVersion: v2
+description: KubeFed helm chart (temp)
+version: 0.0.5
+home: "https://github.com/kubernetes-sigs/kubefed"
+appVersion: "0.0.4"
 maintainers:
-  - name: s12chung
-  - name: alejandroEsc
+- name: alejandroesc
```
Comment (joejulian) [inline staging/kubefed/Chart.yaml:8]: Ugh... unindented lists...
punishment

**Review by joejulian — APPROVED** [review]

**Pull request merged**